addons/web/static/tests/views/list_view_tests.js

```diff
@@ -18570,6 +18570,10 @@ QUnit.module("Views", (hooks) => {
         await clickSave(target);
 
         assert.strictEqual(target.querySelector(".o_field_cell.o_integer_cell").textContent, "321");
+        assert.strictEqual(
+            target.querySelector(".o_list_footer .o_list_number").textContent,
+            "567"
```

Comment:
A bit hard to see where this "567" comes from 😄 

For such comparison, specifying an error that mentions why we need to have this number is very helpful.
Otherwise you end up with "321 is not equal to 567" and... good luck ^^

Reply:
ok, done :)